node/tests/RedisClient.test.ts

```diff
@@ -27,7 +27,7 @@ type Context = {
     client: RedisClient;
 };
 
-const TIMEOUT = 10000;
+const TIMEOUT = 50000;
```

Comment:
It is total IT timeout - could be not enough even in CI

Reply:
What unit of time is used for the timeout?

Reply:
millis, so I updated it 10 to 50 sec
on my VM in runs 9 to 25, so I have a flaky failure